src/collection-view.js

```diff
@@ -165,27 +165,38 @@ Marionette.CollectionView = Marionette.View.extend({
   reorder: function() {
     var children = this.children;
     var models = this._filteredSortedModels();
-    var modelsChanged = _.find(models, function(model) {
+    var modelsAdded = _.find(models, function(model) {
```

Comment:
we're only using this `var` to check if a model has been added so `find` is fine - but perhaps `_.some` would be a better option as that actually returns a `boolean`.